pkg/services/store/testdata/public_testdata.golden.jsonc

```diff
@@ -12,17 +12,17 @@
 //  }
 //  Name: 
 //  Dimensions: 3 Fields by 3 Rows
-//  +----------------------------+--------------------------+---------------+
-//  | Name: name                 | Name: mediaType          | Name: size    |
-//  | Labels:                    | Labels:                  | Labels:       |
-//  | Type: []string             | Type: []string           | Type: []int64 |
-//  +----------------------------+--------------------------+---------------+
-//  | countries.geojson          | application/octet-stream | 255943        |
-//  | example-with-style.geojson | application/octet-stream | 3332          |
-//  | usa-states.geojson         | application/octet-stream | 89263         |
-//  +----------------------------+--------------------------+---------------+
-//  
-//  
+//  +----------------------------+----------------------+---------------+
+//  | Name: name                 | Name: mediaType      | Name: size    |
+//  | Labels:                    | Labels:              | Labels:       |
+//  | Type: []string             | Type: []string       | Type: []int64 |
+//  +----------------------------+----------------------+---------------+
+//  | countries.geojson          | application/geo+json | 255943        |
+//  | example-with-style.geojson | application/geo+json | 3332          |
+//  | usa-states.geojson         | application/geo+json | 89263         |
```

Comment:
This keeps slipping through in a bunch of PRs I've been making and I don't always notice. Is there a reason it keeps changing? Should we `.gitignore` it and use `git add --force` for when we need to change it?

@ryantxu

Reply:
OK, thanks!